### ISSUE TITLE
Uses Accelerate framework instead of the blas package from conda-forge

### ISSUE THREAD

**Issue by mbargull** [conda-forge/slycot-feedstock#23]
Reported by @isuruf in https://github.com/conda-forge/slycot-feedstock/pull/22#issuecomment-578444017 :
> This package is using Accelerate framework instead of the blas package from conda-forge.

See the build logs from https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=115074&view=logs&j=a8dee1cf-d17f-5cb0-b53f-4a93dfdbfb0c&t=aafbbd55-dd7b-5830-1c88-bc5f3db40f31&l=1673 :
```
number of files: 42
Fixing linking of /System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate in /usr/local/miniconda/conda-bld/slycot_1580144896043/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so
New link location is None
Fixing linking of @rpath/libgfortran.4.dylib in /usr/local/miniconda/conda-bld/slycot_1580144896043/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so
New link location is None
Fixing linking of /usr/lib/libSystem.B.dylib in /usr/local/miniconda/conda-bld/slycot_1580144896043/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so
New link location is None
Fixing linking of /usr/local/miniconda/conda-bld/slycot_1580144896043/_build_env/lib/libgcc_s.1.dylib in /usr/local/miniconda/conda-bld/slycot_1580144896043/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so
.. seems to be linking to a compiler runtime, replacing build prefix with host prefix and
.. fixing linking of /usr/local/miniconda/conda-bld/slycot_1580144896043/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/libgcc_s.1.dylib in /usr/local/miniconda/conda-bld/slycot_1580144896043/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so instead
New link location is lib/libgcc_s.1.dylib
Fixing linking of @rpath/libquadmath.0.dylib in /usr/local/miniconda/conda-bld/slycot_1580144896043/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so
New link location is None


   INFO (slycot,lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so): Needed DSO /System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate found in the whitelist
   INFO (slycot,lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so): Needed DSO /usr/lib/libSystem.B.dylib found in the whitelist
   INFO (slycot,lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so): Needed DSO lib/libgfortran.4.dylib found in conda-forge::libgfortran-4.0.0-2
   INFO (slycot,lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so): Needed DSO lib/libgcc_s.1.dylib found in conda-forge::libgfortran-4.0.0-2
   INFO (slycot,lib/python3.8/site-packages/slycot/_wrapper.cpython-38-darwin.so): Needed DSO lib/libquadmath.0.dylib found in conda-forge::libgfortran-4.0.0-2
   INFO (slycot): plugin library package conda-forge::numpy-1.14.6-py38hde6bac1_1201 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (slycot): run-exports library package conda-forge::liblapack-3.8.0-7_h4bb4525_netlib in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (slycot): run-exports library package conda-forge::libcblas-3.8.0-7_h4bb4525_netlib in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (slycot): dso library package conda-forge::python-3.8.1-h5c2c468_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (slycot): run-exports library package conda-forge::libblas-3.8.0-7_h4bb4525_netlib in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
Fixing permissions
```

**Comment by repagh** [comment]
Work is underway to release a new version of slycot. I propose addressing it then. 
Is there a specific preference for yes/no using the openblas vs accelerate framework for OSX? Current recipes in the python-control/Slycot repository now explicitly use Accelerate BLAS.

**Comment by moorepants** [comment]
It could be worth seeing if there is a convention for conda forge packages for this. My vote would be to follow their convention if there is one.

**Comment by repagh** [comment]
I see scipy using plain old blas/cblas and lapack...

**Comment by moorepants** [comment]
Maybe this is the practice:

https://conda-forge.org/docs/maintainer/knowledge_base.html#switching-blas-implementation

i.e. that the users installs whatever versions they want in their environment.

This is how Ubuntu works for example, with configuring alternative packages.

**Comment by moorepants** [comment]
Fixed in #28.
